src/main/java/com/example/repository/BookRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.example.repository;
+
+import com.example.entity.BookEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface BookRepository extends JpaRepository<BookEntity, Long> {
```

Comment:
you should implement BookRepository without by yourself
```suggestion
public interface BookRepository {
```

Reply:
what do u mean? I using JPA it's easy implement or need custom?